knowledge/pop_culture/events/barbenheimer/qna.yaml

```diff
@@ -0,0 +1,24 @@
+created_by: Jason Knott
+domain: movies
+seed_examples:
+- answer: |
```

Comment:
Thank you for your submission; taking a look at your `qna.yaml`, we noticed that you have some issues with the multiline `yaml` syntax.

Can you take a look at: https://yaml-multiline.info/ and fix up the file? Thanks!